### ISSUE TITLE
Problem with deletion of objects

### ISSUE THREAD

**Issue by Luke47** [tristanhimmelman/THContactPicker#44]
Hey!

I really like this library and so far it works like a charm. However, I'm experiencing some unwanted behaviour with iOS 8 / Swift.

Deletion of objects in the view works fine, except for the case when I use the `removeContact` function as it does not seem to remove the object. The bubble stays there.

Do you know how to work around this issue? Thank you!

**Comment by hardikamal** [comment]
@Luke47  Even i m facing the same issue. Were you able to solve it?

**Comment by Luke47** [comment]
@hardikamal 

Yes, I remember that I've made it work. You can check out my fork. I think it was this specific commit:
https://github.com/Luke47/THContactPicker/commit/30e1a3f84ba4ec494eb6701b1a4a06ea6a804ac5

**Comment by hardikamal** [comment]
@Luke47 cool thanks
